### PR TITLE
Add polyfills for IE and safari compatibility

### LINKED ISSUE #225: Does not support IE11.
> As a user using  Internet Explorer 11, when I navigate to signup.steemit.com I should be able to complete a signup.

Currently, presented with completely blank screen.

### PR DESCRIPTION
**Pull request by gregory-latinier**
Fix #225

### CODE REVIEW

**Comment by bonustrack** [comment]
The website is now visible on IE but the logos overlap each others see:
image
Can you fix that @gregory-latinier ?

File: package.json [hunk 1]
```diff
@@ -63,6 +64,7 @@
     "extract-text-webpack-plugin": "1.0.1",
     "file-loader": "0.11.2",
     "history": "3.2.1",
+    "intl": "^1.2.5",
     "jsdom": "9.5.0",
     "json-loader": "0.5.4",
```
Comment (jnordberg) [inline package.json]: Package versions should be absolute

File: views/index.hbs [hunk 2]
```diff
@@ -7,6 +7,7 @@
   <meta id="viewport" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   {{baseCss}}
+  <link rel="stylesheet" href="/css/ie.css" type="text/css" media="all">
```
Comment (jnordberg) [inline views/index.hbs]: Shouldn’t this be in a if ie comment as well?

Comment (gregory-latinier) [reply views/index.hbs]: If IE comments only works up to IE 9 after that the if command is not interpreted anymore

**Pull request merged**